lib/block_view_validator.go

```diff
@@ -0,0 +1,2133 @@
+package lib
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"fmt"
+	"github.com/deso-protocol/core/bls"
+	"github.com/dgraph-io/badger/v3"
+	"github.com/golang/glog"
+	"github.com/holiman/uint256"
+	"github.com/pkg/errors"
+	"io"
+	"math"
+	"net/url"
+	"sort"
+)
+
+//
+// TYPES: ValidatorEntry
+//
+
+type ValidatorEntry struct {
+	ValidatorID   *BlockHash
+	ValidatorPKID *PKID
+	// Note: if someone is updating their ValidatorEntry, they need to include
```

Comment:
All of these fields need good comments.

Reply:
Edit: I put a FIXME at the top for one big mega-comment. Same for block_view_stake.go